packages/material-ui/src/InputBase/InputBase.test.js

```diff
@@ -302,7 +302,7 @@ describe('<InputBase />', () => {
           target: { value: '' },
         });
         assert.strictEqual(handleEmpty.callCount, 1);
-        assert.strictEqual(muiFormControl.onEmpty.callCount, 1);
+        assert.strictEqual(muiFormControl.onEmpty.callCount, 2);
```

Comment:
I have added the ref back after seeing this fail in my env. If you think it's better, I have no objection :).

Reply:
I'm going to do some more investigation.